### DISCUSSION TITLE
Magichome json template RGB

### DISCUSSION THREAD

**Discussion by dicker182** [RavenSystem/esp-homekit-devices#1605] [MEPLHAA Scripts]
Good day,
I am trying to get an working template. The one I found works only with color blue and brightness control. Red and green are dark.

{"c":{"b":[{"g":0,"t":5}]},"a":[{"t":30,"g":[15,13,12,14],"b":[{"g":0}]}]}	

Hope you could help

**Comment by maccoylton** [comment]
Different variants of that controller use different GPIO's, I have three controllers I use currently and two variants

Red:14, Green:5, Blue:12, White:13
Red:12, Green:5, Blue:13, White:15

**Comment by dicker182** [comment]
Thank so far

So I have to do try and error with this for numbers? [15,13,12,14]
Sry, but I am totaly a noob with this templates.

**Comment by dicker182** [comment]
Got it!

{"c":{"b":[{"g":0,"t":5}]},"a":[{"t":30,"g":[14,5,12],"s":1,"b":[{"g":0}]}]}

Thx :)
